tests/test_layers.py

```diff
@@ -1770,6 +1770,26 @@ def model_func(wrapped_import, inputs: torch.Tensor):
   verify_torch_and_convert_to_returnn(model_func, inputs=x, returnn_dummy_input_shape=x.shape)
 
 
+def test_returnn_config_serialization():
+  n_batch, n_feat = 1, 5
+
+  def model_func(wrapped_import, inputs: torch.Tensor):
+    if typing.TYPE_CHECKING or not wrapped_import:
+      import torch
+    else:
+      torch = wrapped_import("torch")
+    arange = torch.arange(inputs.shape[1])
+    return inputs + torch.reshape(arange, (1, -1))
+
+  rnd = numpy.random.RandomState(42)
+  x = rnd.normal(0., 1., (n_batch, n_feat)).astype("float32")
+  converter = verify_torch_and_convert_to_returnn(model_func, inputs=x)
+
+  cfg = converter.get_returnn_config_serialized()
+  print(f"Serialized config:\n\n{cfg}")
+  exec(cfg)
```

Comment:
I would further extend this to also try to load it again via RETURNN. The result of `exec` can be put into a new `cfg` object. See `config_net_dict_via_serialized` and then `dummy_run_net` in the test cases of returnn-common.

Reply:
I added that. We have to adapt the training dataset though. Did you want to do training here also? Or just forwarding as in the other tests?